### PR TITLE
Backport from release/ipm 2.4.0

### PR DESCRIPTION
**Pull request by aquette**

### CODE REVIEW

File: ipm/2.4.0.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Eaton Intelligent Power Manager Editions 2.4.0 - Release note 
+
+*December 2021*
```
Comment (eric3873) [inline ipm/2.4.0.md:3]: Not the same date format

File: ipm/2.4.0.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Eaton Intelligent Power Manager Editions 2.4.0 - Release note
```
Comment (eric3873) [inline ipm/2.4.0.md:1]: Note the same title format (as 2.3.1)

File: ipm/2.4.0.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Eaton Intelligent Power Manager Editions 2.4.0 - Release note 
+
+*December 2021*
+
+## Scope
```
Comment (eric3873) [inline ipm/2.4.0.md:5]: : missing at the end of § (or remove previous in 2.3.1)

**Review by eric3873 — APPROVED** [review]

File: ipm/2.4.0.md (hunk 2 above)
Comment (aquette) [reply ipm/2.4.0.md:1]: applied this one to others

File: ipm/2.4.0.md (hunk 1 above)
Comment (aquette) [reply ipm/2.4.0.md:3]: applied the same as others (Month Year)

File: ipm/2.4.0.md (hunk 3 above)
Comment (aquette) [reply ipm/2.4.0.md:5]: applied the : to 2.4.0 too

**Pull request closed without merging**